src/app/modules/entry-live/views/live-discovery-table/devices-table/live-discovery-devices-table.config.ts

```diff
@@ -61,7 +61,7 @@ export class LiveDiscoveryDevicesTableConfig extends ReportDataBaseConfig {
             sortOrder: 2,
           },
           'view_unique_buffering_users': {
```

Comment:
@diamond-darrell - metric name should probably be view_buffer_time_ratio

Reply:
fixed